doc/test-framework/scorecard.md

```diff
@@ -168,9 +170,10 @@ Following the description of each internal [Plugin](#plugins). Note that are 8 i
 
 | Test        | Description   |
 | --------    | -------- |
+| Bundle Validation | This test validates the bundle manifests found in the bundle directory as specifed by the bundle flag.  If the bundle contents contain errors, then the test result output will include the validator log as well as error messages from the validation library.|
```

Comment:
```suggestion
| OLM Bundle Validation | This test validates the OLM bundle manifests found in the bundle directory as specified by the bundle flag.  If the bundle contents contain errors, then the test result output will include the validator log as well as error messages from the validation library. See more info; [here]<URL for the community doc which explains/specified what a bundle is>|
```

Reply:
I took your suggestion and added a link to the bundle documentation from operator-registry's docs.